cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -93,6 +93,10 @@ message Options {
 	// logrus log levels: "trace", "debug", "info", "warn", "error", "fatal", "panic". This setting will override
 	// the `debug` field if both are specified, unless the level specified is also "debug", as these are equivalent.
 	string log_level = 16;
+
+	// no_inherit_host_timezone specifies to skip inheriting the hosts time zone for WCOW UVMs and instead default to
+	// GMT.
+	bool no_inherit_host_timezone = 17;
```

Comment:
I'm not sure. If down the line we change the default to UTC for some reason, then we need to deprecate this solely based off the name instead of just updating the comment. Might be a bit odd if a clients machine(s) were already in gmt and then have to have the thought of "wait weren't my containers using gmt already and I don't have this on" also but this is likely overthinking Danny talking on the latter bit.

Reply:
Not to rehash this, but if the container is in GMT, it will transition to Daylight saving, and that can be unexpected for end users, especially if they transition to Daylight savings at a different date than GMT.